app/views/messages/index.html.erb

```diff
@@ -0,0 +1,76 @@
+<div class="container">
+  <%= link_to 'Home', root_path %>
+</div>
+
+<div class="container">
+<div class="row">
+<div class="col-8">
+  <h3>Messages</h3>
+  <div class="over">
+    <% @messages.each do |message| %>
+    <% if message.sender_id == current_user.id || message.recipient_id == current_user.id %>
+    <% if message.sender_id == current_user.id %>
+      <% recipient = message.recipient %>
+    <% else %>
+      <% recipient = message.sender %>
+    <% end %>
+    <table class="table tsize">
+        <col width="25%">
+        <col width="65%">
+        <col width="5%">
+        <col width="5%">
+        <tr>
+          <td><%= recipient.first_name %> <%= recipient.last_name %></td>
+          <td>
+            <% message.contents.each do |content| %>
+                <% if content.equal?(message.contents.last)  %>
+                  <%= content.body %>
+                <% end %>
+            <% end %>
+          </td>
+          <td>
+            <% if !message.unread_content_count(current_user).zero? %>
+              <%= message.unread_content_count(current_user) %>
+            <% end %>
+          <td>
+          <td><%= link_to '', message_contents_path(message), :class => "fas fa-pen-nib fa-xm" %></td>
+          <td><%= link_to '', message_path(message), method: :destroy, :class => "fas fa-trash-alt fa-xm" %></td>
```

Comment:
The text for both of these links are blank. Is that intentional?

Reply:
Yes, I made empty strings for  using font(icon).